cluster/manifests/skipper/deployment.yaml

```diff
@@ -578,7 +579,6 @@ spec:
           - '-kubernetes-annotation-filters-append={{ .Cluster.ConfigItems.skipper_kubernetes_annotation_filters_append }}'
           - '-kubernetes-east-west-range-annotation-predicates={{ .Cluster.ConfigItems.skipper_kubernetes_east_west_range_annotation_predicates }}'
           - '-kubernetes-east-west-range-annotation-filters-append={{ .Cluster.ConfigItems.skipper_kubernetes_east_west_range_annotation_filters_append }}'
-          - "-reverse-source-predicate"
```

Comment:
We'd need to also also drop `-whitelisted-healthcheck-cidr`, see https://github.com/zalando-incubator/kubernetes-on-aws/pull/9308